cssdb.json

```diff
@@ -128,7 +149,25 @@
     "polyfills": [
       {
         "type": "PostCSS Plugin",
-        "link": "https://github.com/jonathantneal/postcss-color-mod-function"
+        "link": "https://github.com/csstools/postcss-color-mod-function"
+      }
+    ]
+  },
+  {
+    "id": "container-queries",
+    "title": "Container Queries",
+    "description": "New container property and container at rule to make changes depending on the container's size",
+    "specification": "https://drafts.csswg.org/css-contain-3/#container-queries",
+    "stage": 0,
+    "caniuse": "css-container-queries",
+    "example": ".container {\n  contain: layout inline-size;\n}\n\n@container (min-width: 700px) {\n  .container {\n    /* styles applied when a container is at least 700px */\n  }\n}",
+    "docs": {
+      "mdn": "https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Container_Queries"
+    },
+    "polyfills": [
```

Comment:
Should we consider : https://www.npmjs.com/package/container-query-polyfill

Maybe this is a list where multiple polyfills can be added side by side? I don't know the format that well :)

Reply:
Great point!